plato/draw/Box.py

```diff
@@ -0,0 +1,56 @@
+import itertools
+import numpy as np
+from .Lines import Lines
+from .. import math
+from .internal import attribute_setter, ShapeDecorator, ShapeAttribute
+
+@ShapeDecorator
+class Box(Lines):
+    """Create a box primitive to draw the frame
+
+    Example: prim = draw.Box((Lx,Ly,Lz,xy,xz,yz), width, color)
```

Comment:
Can you follow this style so it renders properly in the docs? See how it uses two colons, a blank line, and an indentation.

https://github.com/glotzerlab/plato/blob/f48f04b33e6a6074caed3a9e82aca9934eb37755/plato/draw/Arrows2D.py#L16-L19

Reply:
Style applied